upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -837,7 +837,7 @@ func (b *BootstrapChannelBuilder) buildAddons(c *fi.ModelBuilderContext) (*chann
 			"k8s-1.7":    "2.6.12-kops.1",
 			"k8s-1.7-v3": "3.8.0-kops.2",
 			"k8s-1.12":   "3.9.6-kops.1",
-			"k8s-1.16":   "3.16.1-kops.2",
+			"k8s-1.16":   "3.16.2-kops.2",
```

Comment:
```suggestion
			"k8s-1.16":   "3.16.1-kops.3",
```

Reply:
@hakman can you tell me what does `-kops.x` suffix means here ? also, i've changes the version of calico images from 3.16.1 to 3.16.2

Reply:
Yes, please revert the Calico version change.
`-kops.x` means the manifest version. It is used internally to determine if the manifest changed and needs to be reapplied.